microprofile/WFLY-19954_Preview_Support_vertx_feature_pack.adoc

```diff
@@ -0,0 +1,231 @@
+---
+categories:
+  - microprofile
+stability-level: preview
+issue: https://github.com/wildfly/wildfly-proposals/issues/662
+feature-team:
+ developer: gaol
+ sme:
+  - kabir
+  - fl4via
+  - jasondlee
+ outside-perspective:
+  -
+promotes:
+promoted-by:
+---
+
+= Integrate wildfly-vertx-feature-pack to WildFly Preview
```

Comment:
Just a small comment about terminology here, WildFly Preview is itself a feature pack - is this proposal more about adding the Vert.x subsystem / extension to the WildFly Preview Feature Pack?

Reply:
Yes, it is.  I thought WildFly Preview is a  different distribution, so `Adding the vertx subsystem / extension to the WildFly Preview Feature Pack` would be more accurate, will update it. :)